### PR TITLE
Register page

### PR DESCRIPTION
**Pull request by Vishal-raj-1**
- [x] First name and last name moved to a single row.
- [x] Change label from Cluster Location -> Cluster Region. Add help text: (Choose a region nearest to your warehouse region)
- [x] Cluster shouldn't be selected by default

If you don't select any cluster location then it will give you an error notification (Check below image):

error

### CODE REVIEW

File: webapp/app/common/enums/AppCluster.ts [hunk 1]
```diff
@@ -1,9 +1,11 @@
 export enum AppCluster {
+  SELECT = "SELECT",
```
Comment (mkstayalive) [inline webapp/app/common/enums/AppCluster.ts:2]: SELECT = "",

File: webapp/pages/auth/register.tsx [hunk 2]
```diff
@@ -64,25 +64,27 @@ function Register(props: serverSideProps) {
           lastName: "",
           password: "",
           confirmPassword: "",
-          clusterLocation: AppCluster.US,
+          clusterLocation: AppCluster.SELECT,
```
Comment (mkstayalive) [inline webapp/pages/auth/register.tsx:67]: should be empty

File: webapp/app/common/enums/AppCluster.ts (hunk 1 above)
Comment (mkstayalive) [inline webapp/app/common/enums/AppCluster.ts:2]: Shouldn't have a dummy value

**Comment by Vishal-raj-1** [comment]
Solved in #135

**Pull request closed without merging**